Source/ThirdParty/ANGLE/ANGLE.xcodeproj/project.pbxproj

```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 55;
+	objectVersion = 54;
```

Comment:
Do we know what causes this unnecessary churn on this property?

Reply:
I think it's different xcode versions, but I thought mine was up-to-date.